tests/test_tools.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env pytest -vs
+"""Tests for Validation functions."""
+
+# Third-Party Libraries
+import pytest
+
+# cisagov Libraries
+from tools.gophish_complete import get_campaign_id
+
+
+class TestComplete:
+    """Test gophis-complete script."""
+
+    @pytest.mark.parametrize(
+        "campaigns", [{"1": "RV0000-C1", "2": "RV0000-C2", "3": "RV0000-C3"}]
+    )
+    def test_get_campaign_id_found(self, campaigns):
+        """Test finding campaign id by campaign name when successful."""
```

Comment:
I'd reword this a bit:  `"""Verify correct campaign id is returned when a valid campaign name is provided."""`

Reply:
Changed wording in commit 517e478.